oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -637,13 +636,16 @@ int mca_spml_ucx_ctx_create(long options, shmem_ctx_t *ctx)
 
 void mca_spml_ucx_ctx_destroy(shmem_ctx_t ctx)
 {
+    mca_spml_ucx_ctx_t *ucx_ctx = (mca_spml_ucx_ctx_t *)ctx;
     MCA_SPML_CALL(quiet(ctx));
 
     SHMEM_MUTEX_LOCK(mca_spml_ucx.internal_mutex);
     _ctx_remove(&mca_spml_ucx.active_array, (mca_spml_ucx_ctx_t *)ctx);
     _ctx_add(&mca_spml_ucx.idle_array, (mca_spml_ucx_ctx_t *)ctx);
     SHMEM_MUTEX_UNLOCK(mca_spml_ucx.internal_mutex);
 
+    ucp_worker_flush(ucx_ctx->ucp_worker);
```

Comment:
I did not notice that quiet(ctx) is called in the beginning of this function, so it makes no sense to call flush here then.
I'm sorry for this confusion.

Reply:
yep, my fault. in current implementation of quiet there is flush_nb is called and it has own progress loop for worker (independent from opal progress).

Reply:
ok I'll remove it

Reply:
done, removed flush